.github/workflows/build.yml

```diff
@@ -24,6 +24,8 @@ jobs:
           - nrfmicro_13
           - proton_c
         shield:
+          - architeuthis_dux_left
```

Comment:
We are no longer adding to the `build.yml` in new shields, since it causes merge conflicts. We'll eventually drive the build from the metadata files as well.

Reply:
Ah, I thought so. Thanks. I've now removed the changes to `build.yml` from this PR.

I've also just raised a small PR aligning the New Shield documentation: https://github.com/zmkfirmware/zmk/pull/952